apache/zeppelin
# [ZEPPELIN-3035] Ability to display the same result multiple times for each paragraph.

### What is this PR for?
Sometimes we need to have multiple visualizations with each result of the query. Currently, we can only do it by copying query which increases the load on the server.
There should be a way to specify the number of result copies we want and the results should be duplicated in the UI.


### What type of PR is it?
[ Feature | Documentation ]

### Todos
* [Done] - Adding the abilty to specify the number of results in UI.
* [Done] - Adding documentations for the same. 

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3035

### How should this be tested?
* Open any new notebook and select the num results options in paragraph settings. 
* Run paragraph and see the results. 

### Screenshots (if appropriate)


![image1](https://user-images.githubusercontent.com/858038/32418661-f7e8c1ec-c265-11e7-81fa-04c427a43abe.png)

<img width="1438" alt="multiple_result_visualization" src="https://user-images.githubusercontent.com/858038/32418670-fcf49c1a-c265-11e7-8d9e-8dcfe546e25f.png">



### Questions:
* Does the licenses files need update?
   No
* Is there breaking changes for older versions?
   No
* Does this needs documentation? 
   Yes.
